point_cloud_client/src/lib.rs

```diff
@@ -19,12 +21,27 @@ impl PointCloudClient {
                     .map(|octree| *octree)
             })
             .collect();
+        let mut aabb = Aabb3::zero();
+        if let Ok(octrees) = &octrees {
+            if let Some(first_octree) = octrees.first() {
+                aabb = *first_octree.bounding_box();
+                for octree in octrees.iter().skip(1) {
+                    let octree_bbox = octree.bounding_box();
+                    aabb = aabb.grow(octree_bbox.min()).grow(octree_bbox.max());
```

Comment:
I also saw recently that there is the method union() to merge two Aabbs

Reply:
Thanks, I modified it.